examples/flutter_gallery/lib/demo/material/cards_demo.dart

```diff
@@ -45,116 +54,303 @@ final List<TravelDestination> destinations = <TravelDestination>[
+  const TappableTravelDestinationItem({ Key key, @required this.destination, this.shape })
+    : assert(destination != null),
+      super(key: key);
+
+  // This height will allow for all the Card's content to fit comfortably within the card.
+  static const double height = 312.0;
+  final TravelDestination destination;
+  final ShapeBorder shape;
 
+  @override
+  Widget build(BuildContext context) {
     return SafeArea(
       top: false,
       bottom: false,
-      child: Container(
+      child: Padding(
         padding: const EdgeInsets.all(8.0),
-        height: height,
-        child: Card(
-          clipBehavior: Clip.antiAlias,
-          shape: shape,
-          child: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: <Widget>[
-              // photo and title
-              SizedBox(
-                height: 184.0,
-                child: Stack(
-                  children: <Widget>[
-                    Positioned.fill(
-                      child: Image.asset(
-                        destination.assetName,
-                        package: destination.assetPackage,
-                        fit: BoxFit.cover,
+        child: Column(
+          children: <Widget>[
+            const SectionTitle(title: 'Card that can be tapped'),
+            SizedBox(
+              height: height,
+              child: Card(
+                // This ensures that the Card's children (including the ink splash) are clipped correctly.
+                clipBehavior: Clip.antiAlias,
+                shape: shape,
+                child: InkWell(
+                  onTap: () {
+                    print('Card was tapped');
+                  },
+                  splashColor: Theme.of(context).colorScheme.primary.withAlpha(30),
+                  child: TravelDestinationContent(destination: destination),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class SelectableTravelDestinationItem extends StatefulWidget {
+  const SelectableTravelDestinationItem({ Key key, @required this.destination, this.shape })
+    : assert(destination != null),
+      super(key: key);
+
+  final TravelDestination destination;
+  final ShapeBorder shape;
+
+  @override
+  _SelectableTravelDestinationItemState createState() => _SelectableTravelDestinationItemState();
+}
+
+class _SelectableTravelDestinationItemState extends State<SelectableTravelDestinationItem> {
+
+  // This height will allow for all the Card's content to fit comfortably within the card.
+  static const double height = 312.0;
+  bool _isSelected = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      top: false,
+      bottom: false,
+      child: Padding(
+        padding: const EdgeInsets.all(8.0),
+        child: Column(
+          children: <Widget>[
+            const SectionTitle(title: 'Card that can be selected'),
```

Comment:
Some people (me) might not realize that a long press is needed here. Maybe:       

const SectionTitle(title: 'Card that can be selected (long press)'),

Reply:
done